### ISSUE TITLE
Dungeon art

### ISSUE THREAD

**Issue by tukkek** [tukkek/javelin#169]
The one Dungeon type has been separated into 4 subtypes for Javelin 1.7 - this was done to make dungeons much more varied but also to better populate the game world now that most locations are built by the towns, instead of being found around the map. 

The images being used for the 4 subtypes (Caves, Dungeons, Ruins and Keeps) are libre images found elsewhere and Javelin would benefit a lot from having new images for them. The reasons include some of these images being resized or hacked to fit the 32x32p format and also the fact that their power order was chosen based on the available images (which ones feel more "imposing" and "powerful" in the game screen). Ideally, the correct order would be:

* Caves: smallest dungeons, pretty easy
* Keeps: small-to-medium dungeons
* Ruins: medium-to-big dungeons (can get quite big)
* Dungeons: full-fledged, potentially massive dungeons (possibly the hardest challenges in the game!)

It is imperative that when creating new images, their appearance and size reflect that power scale on the world screen - immediately allowing the player to guess that a Dungeon is much more dangerous to visit than a Cave (but the game also presents this information on entering and hovering the mouse as well).

**Comment by tukkek** [comment]
Later on, each subtype could also have new images for their floor and walls inside (so another 2 images per subtype) - but that's not as important as having new location images to better represent them in the game world.

**Comment by tukkek** [comment]
Here are the current images, for reference:

~~Cave ![locationcave](https://user-images.githubusercontent.com/3819826/38892354-60e812de-425d-11e8-8acf-4a4ced555be8.png)~~
Dungeon ![locationdungeon](https://user-images.githubusercontent.com/3819826/38892356-610c8f24-425d-11e8-93db-1b6cd485fd45.png)
Ruins ![locationruins](https://user-images.githubusercontent.com/3819826/38892359-614ea8aa-425d-11e8-8c65-7310fd59ce6c.png)
Keep ![locationkeep](https://user-images.githubusercontent.com/3819826/38892357-612e2166-425d-11e8-8d45-685261a08b00.png)

The order of size, for new images, would be: Cave, Keep, Ruin, Dungeon (from smallest to biggest).

**Comment by ThePizzaFrank** [comment]
![locationcave](https://user-images.githubusercontent.com/13767119/91868125-684b8d80-ec42-11ea-8fa3-38435b23bf20.png)
Cave

**Comment by tukkek** [comment]
Hey Pizza Frank is that you? Long time, how you've been in 2020?! Sorry I haven't been on Discord much for a while now...

I've added the cave on commit https://github.com/tukkek/javelin/commit/29575cb894471ca8f7d2d6a6a101f9d867a8f5d9 (it's on the 1.7 development branch). Thanks a bunch mate!

If it's not too much trouble, could you later on try darkening the "entrance" part a bit, closer to black? Depending on resolution/zoom level might make it more apparent that it's a cave :) I'm thinking people might think it's a rock if it's too small...

**Comment by ThePizzaFrank** [comment]
Yup its me.  Been doing well despite all of 2020's quirks, though of javelin recently so I decided to make more stuff.  I'll see if I can make the entrance a bit darker, or maybe lighten the outside rocks.

**Comment by ThePizzaFrank** [comment]
![locationcave](https://user-images.githubusercontent.com/13767119/91898249-719c2080-ec69-11ea-830f-eb61ab510179.png)

**Comment by ThePizzaFrank** [comment]
![locationdungeon](https://user-images.githubusercontent.com/13767119/96390760-217b0e00-1184-11eb-81e7-ee12711095ce.png)
for the dungeon
![locationdungeon2](https://user-images.githubusercontent.com/13767119/96612968-6742ee00-12cc-11eb-819b-e2851e909dab.png)
also heres a light version as well if it fits better

**Comment by ThePizzaFrank** [comment]
![locationkeep](https://user-images.githubusercontent.com/13767119/96653095-48f9e400-1306-11eb-9a31-c033db101e16.png)
Another one, this is the keep

**Comment by ThePizzaFrank** [comment]
![locationruins](https://user-images.githubusercontent.com/13767119/96762182-50bd9500-13a7-11eb-8c88-60a10d419bc0.png)
And finally, ruins.  I wanted to add more to this but every time it would look too messy

**Comment by tukkek** [comment]
Oh wow, I seem to have missed your comments here! Glad to know you're holding up in these trying times! 

Here's a picture of your new images in-game:

![Screenshot_20201024_161808](https://user-images.githubusercontent.com/3819826/97091764-a8ad0380-1614-11eb-945c-24a37ab55269.png)

I've taken the liberty to adjust the brightness and contrast on some of them to make them as similar as possible color-wise (as you can see in the screenshot). Since they are all different versions of the same feature (Dungeons), it is important that the player be able to identify them as "the same thing" visually as well, as much as possible :)

![locationcave](https://user-images.githubusercontent.com/3819826/97091820-11947b80-1615-11eb-9b6d-733e30e63847.png) ![locationdungeon](https://user-images.githubusercontent.com/3819826/97091822-15280280-1615-11eb-90b5-bb1f72ccf335.png) ![locationruins](https://user-images.githubusercontent.com/3819826/97091825-1b1de380-1615-11eb-923b-0d95119e11b5.png) ![locationkeep](https://user-images.githubusercontent.com/3819826/97091828-1e18d400-1615-11eb-89e9-583de4b20634.png)

If you don't mind me taking the liberty, I'm very happy with the results - but if you wanna do it yourself for whatever reason, by all means go for it!

I'll be honest with you: I've been thinking of removing one of the Dungeon tiers since I'm now suspecting 4 dungeon tiers plus the Temples is a bit of an overkill. Right now I'm thinking I'll remove Ruins and redesign the Temples to be the "fourth dungeon tier" but in that case I'll keep your image around and think of something else it could be used for, if that's alright with you.

Thanks so much for remembering the project and helping me out yet once more! If you feel like contributing more work, I can tag or assign you to other issues I've been thinking should be some of the things I'll be working on next!
